### PR TITLE
ProjectService_AddUserToProjectChanges

### PR DESCRIPTION
**Pull request by mekk1t**
Task:

- Add new properties

- Delete DbProjectManagerUser.cs

- Change implementation of the "Add user to project" -- WARNING! The task "Add user to project" was not even started, so i cannot make changes to that.

### CODE REVIEW

File: DigitalOffice/Services/ProjectService/Database/Entities/DbProject.cs [hunk 1]
```diff
@@ -13,7 +13,6 @@ public class DbProject
         public Guid DepartmentId { get; set; }
         public bool Deferred { get; set; }
         public bool IsActive { get; set; }
-        public ICollection<DbProjectManagerUser> ManagersUsersIds { get; set; }
         public ICollection<DbProjectWorkerUser> WorkersUsersIds { get; set; }
```
Comment (KostinP) [inline DigitalOffice/Services/ProjectService/Database/Entities/DbProject.cs:16]: New migration?

Comment (mekk1t) [reply DigitalOffice/Services/ProjectService/Database/Entities/DbProject.cs:16]: Yeah, will add.

My bad, working in a stressful environment

Comment (KostinP) [reply DigitalOffice/Services/ProjectService/Database/Entities/DbProject.cs:16]: Write which team you will succeed with. I tried through "Add-Migration {Name} -Namespace {Namespace}", but it didn't work out for me.

Comment (mekk1t) [reply DigitalOffice/Services/ProjectService/Database/Entities/DbProject.cs:16]: @KostinP you mean "command" not team, Pasha xD
I committed changes and wrote how-to there.
In my case, there were already some migrations in Database/Migrations/, so the standard command worked out.

Comment (KostinP) [reply DigitalOffice/Services/ProjectService/Database/Entities/DbProject.cs:16]: I even stopped looking at what the translator gives me :D

**Review by KostinP — APPROVED** [review]

**Comment by KostinP** [comment]
Conflicts need to be resolved.

**Review by lalagvana — APPROVED** [review]

**Comment by mekk1t** [comment]
Errors building Tests project.
Huh, it did not come up somehow yesterday.

Will fix tests, by removing ManagerUserId usage and adding new properties to test and validate (if needed)

**Comment by mekk1t** [comment]
image

**Review by ifelseelif — APPROVED** [review]

**Review by djino97 — APPROVED** [review]

**Review by MihailKulikov — APPROVED** [review]

**Pull request merged**